### PR TITLE
Add issue template for improvement suggestions

### PR DESCRIPTION
**Pull request by GANGANI**
Add issue template for improvement suggestions. The added issue templates can try out using [sample repo](https://github.com/GANGANI/Sample-Issue-Template/issues/new/choose).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: ✅ Improvement suggestion
+about: Suggest an improvement for the project
+title: '[IMPROVEMENT]'
+labels: 'Type/Improvement'
+assignees: ''
+
+---
+
+**Is your suggestion related to a experience ? Please describe.**
```
Comment (sherenewso2) [inline .github/ISSUE_TEMPLATE/improvement.md:10]: a experience ? --> an experience?

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<!-- A clear and concise description of what the suggestion is. Ex. Need to improve experience in [...] -->
```
Comment (sherenewso2) [inline .github/ISSUE_TEMPLATE/improvement.md:11]: A clear and concise description of what the suggestion is (e.g., Need to improve experience in [...])

File: .github/ISSUE_TEMPLATE/improvement.md (hunk 1 above)
Comment (GANGANI) [reply .github/ISSUE_TEMPLATE/improvement.md]: Fixed https://github.com/wso2/product-is/pull/8055/commits/1165aa490b71d547eeb8a5d263ff5453d64bf03d

File: .github/ISSUE_TEMPLATE/improvement.md (hunk 2 above)
Comment (GANGANI) [reply .github/ISSUE_TEMPLATE/improvement.md]: Fixed https://github.com/wso2/product-is/pull/8055/commits/1165aa490b71d547eeb8a5d263ff5453d64bf03d

**Pull request merged**